client/src/Movies/SavedList.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 const SavedList = props => (
   <div className="saved-list">
     <h3>Saved Movies:</h3>
     {props.list.map(movie => (
       <span className="saved-movie">{movie.title}</span>
     ))}
+    <Link to= '/movies'>
```

Comment:
Nice use of the Link. You Should look into `<NavLink>` and know the difference between NavLink and Link